Assets/MixedRealityToolkit.SDK/Experimental/Features/Utilities/WorldAnchorManager.cs

```diff
@@ -0,0 +1,604 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+#if UNITY_WSA
+using System;
+using System.Collections.Generic;
+using UnityEngine.XR.WSA;
+using UnityEngine.XR.WSA.Persistence;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Utilities
+{
+    /// <summary>
+    /// Wrapper around world anchor store to streamline some of the persistence API busy work.
+    /// </summary>
+    public class WorldAnchorManager : MonoBehaviour
```

Comment:
Re: removal of Singleton

Was there a reason this wasn't adapted into something like an extension service?

Reply:
My goal here was to get a straight, minimal port in to unblock folks who needed it for historical reasons.

Reply:
Looking at the code (not just the comments :) ), this is in the experimental namespace and should be safe to deliver initially as a monobehaviour to get developer feedback.